src/components/AddBooks.js

```diff
@@ -1,11 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux/';
+import { addbook } from '../redux/books/book';
 import './Addbooks.css';
 
-const AddBooks = () => (
-  <form className="form">
-    <input type="text" placeholder="Title" name="title" />
-    <input type="text" placeholder="Author" name="author" />
-    <button type="submit">Submit</button>
-  </form>
-);
+const AddBooks = () => {
+  const dispatch = useDispatch();
+
+  const [input, setInput] = useState({
+    title: '',
+    author: '',
+  });
+  const [books] = useState();
+  console.log(books, setInput);
```

Comment:
- [ ] It's not a good practice to leave your console.log in the code so please try to remove it